src/commands/mouseDown.ts

```diff
@@ -42,9 +48,9 @@ export async function realMouseDown(
     x,
     y,
     clickCount: 1,
-    buttons: 1,
+    buttons: mouseButtonNumbers[options.button ?? "left"],
```

Comment:
btw. configuring `buttons` is not necessary unless we have **additional** buttons pressed down. It's not possible to configure this right now so this could be omitted - but I'd like to add that feature in the future, so it's convenient to leave this here for the time being.